service/worker/indexer/esProcessor.go

```diff
@@ -85,7 +85,7 @@ func NewESProcessorAndStart(config *Config, client *elastic.Client, processorNam
 		metricsClient: metricsClient,
 	}
 
-	processor, err := client.BulkProcessor().
+	processor, err := client.GetRawClient().BulkProcessor().
```

Comment:
Do not expose the entire raw client.  You can just expose BulkProcessor.

Reply:
updated